---
permalink: /research/
title: "Research"
classes: wide 
---

<h2>Quantitative genetics</h2>
<p>To what extent can we predict quantitative phenotypes based on genotype? Mutations affect phenotype by perturbing the underlying cellular network: they can alter 1) enzymatic quality of proteins, 2) gene expression, and 3) protein folding amongst others. 
Focusing on the genetic basis of traits, we are broadly interested in asking what are the molecular processes that determine differences between individuals and therefore are acted on by evolutionary forces.
</p>
<h6><i class='fas fa-dna'></i> Systems biology of complex traits</h6>
<p>
Thousands of polymorphisms in populations are responsible for phenotypic differences between individuals and between species. Which 'molecular knobs' are affected and are the main players between phenotypic divergence? Our research uses extremely large panels of derived strains from the wild to get at the systems biology of the cell.
</p>
<h6><i class='fas fa-briefcase-medical'></i> Precision medicine</h6>
<p>Our lab is participating in the effort to systematically characterize all functional missense variation in human genes. Our goal is to determine disease risks such as cancer <i>before</i> manifestation of symptoms (shifting medicine to a preventive science), and to reveal genetic determinants of inter-personal differences in molecular pathology to design better treatments,  
</p>
<h3>Relevance</h3>
<p>We hope to develop machine learning models of the cell to bring new insight into biological diseases. Examples of current projects aim to develop technologies for regenerative medicine in addressing aggregation-prone proteins found in Parkinson's and for screening disregulations that lead to cancers.
  </p>
<h2>Evolution</h2>
<p>Comparative genomics is the most popular approach to studying evolution. In contrast, our approach is to watch evolution as it happens, in the context of experiments that are performed under controlled conditions. We are interested in the dynamical process of selection and the evolutionary forces behind phenotypic changes. Ultimately, we hope to leverage our understanding of these processes to predict evolution.</p>
<h6><i class='fas fa-flask'></i> Phenotypic evolution</h6>
<p>Our lab makes use of several <a href="http://myxo.css.msu.edu/index.html" target="_blank">experimental evolution experiments <i class="fas fa-external-link-alt"></i></a> to dissect the cellular basis of adaptation. For example, we know that transcriptional regulation plays a major role in cellular differentiation. However, how <i>evolution</i> tinkers with regulatory networks is less clear.</p>
<h6><i class='fas fa-syringe'></i> Vaccinology</h6>
<p>B-cells undergo a selection process in germinal centers akin to evolutionary dynamics in microbial populations. A quantitative modeling of these processes is critical for developing an understanding of the repeatability of this evolutionary system and how it can be applied to vaccination success.
</p>
<p>
Our lab aims to understand how vaccine design is related to the underlying evolutionary process behind protective B-cell lineages.
</p>

<h3>Relevance</h3>
<p>
Our work in laboratory environments allows us to determine the relevant parameters of evolution, which can help in understanding and forecasting evolution in the real world. For example, note the similarity between the evolution of the Sars-CoV-2 variants of concern from <a href='https://www.medrxiv.org/content/10.1101/2020.12.24.20248822v1' target='_blank'>Davies et al, 2020<i class='fas fa-external-link-alt'></i></a> (left), when compared to <a href='https://www.nature.com/articles/s41586-019-1749-3' target='_blank'>high-resolution lineage tracking<i class='fas fa-external-link-alt'></i></a> of yeast populations in a flask (right) : </p>
<img src="/assets/images/sars_vs_yeast.png" alt="" style="max-height:300;max-width:400" />
